docs/VictoriaLogs/Roadmap.md

```diff
@@ -6,18 +6,16 @@ It isn't recommended migrating from existing solutions to VictoriaLogs Preview y
 
 The following functionality is available in VictoriaLogs Preview:
 
-- [Data ingestion](https://docs.victoriametrics.com/VictoriaLogs/#data-ingestion).
+- [Data ingestion](https://docs.victoriametrics.com/VictoriaLogs/ingestion.html).
 - [Querying](https://docs.victoriametrics.com/VictoriaLogs/#querying).
 - [Querying via command-line](https://docs.victoriametrics.com/VictoriaLogs/#querying-via-command-line).
 
-See [operation docs](https://docs.victoriametrics.com/VictoriaLogs/#operation) for details.
+See [operation docs](https://docs.victoriametrics.com/VictoriaLogs/quickstart.html) for details.
```

Comment:
should we change the title from operation docs to quickstart? just not to confuse people but still mention the purpose